data/pom.yml

```diff
@@ -11,70 +11,59 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
 kieWarsDescription: WAR files for all supported containers
 
 latestFinal:
-    version: 7.61.0.Final
-    releaseDate: 2021-11-04
+    version: 7.62.0.Final
+    releaseDate: 2021-11-29
 
-    droolsZip: https://download.jboss.org/drools/release/7.61.0.Final/drools-distribution-7.61.0.Final.zip
-
-    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.61.0.Final/droolsjbpm-integration-distribution-7.61.0.Final.zip
-
-    businessCentralEAP7WAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-eap7.war
-    businessCentralWildFlyWAR: https://download.jboss.org/drools/release/7.61.0.Final/business-central-7.61.0.Final-wildfly23.war
+    droolsZip: https://download.jboss.org/drools/release/7.62.0.Final/drools-distribution-7.62.0.Final.zip
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.62.0.Final/droolsjbpm-integration-distribution-7.62.0.Final.zip
+    businessCentralWildFlyWAR: https://download.jboss.org/drools/release/7.62.0.Final/business-central-7.62.0.Final-wildfly23.war
+    kieExecutionServerZip: https://download.jboss.org/drools/release/7.62.0.Final/kie-server-distribution-7.62.0.Final.zip
+    kieWARS: https://repo1.maven.org/maven2/org/kie/server/kie-server/7.62.0.Final/
 
     # as part of the migration to JBAke we have a regex based script to update the version
     # as the below is hardcoded to a fixed version, we just inline the value in the content pages for now, so the regex based script won't change them
     # droolsjbpm-tools iss not released any more, last stable version is 7. 46.0.Final
     # latestDroolsJbpmToolsZip: https://download.jboss.org/drools/release/7. 46.0.Final/droolsjbpm-tools-distribution-7. 46.0.Final.zip
 
-    # droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.61.0.Final/droolsjbpm-tools-distribution-7.61.0.Final.zip
-
-    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.61.0.Final/drools-docs/html_single/index.html
+    # droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.62.0.Final/droolsjbpm-tools-distribution-7.62.0.Final.zip
 
-    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.61.0.Final/kie-api-javadoc/index.html
+    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.62.0.Final/drools-docs/html_single/index.html
```

Comment:
Can you kindly explain these YAML `key` changes?

Reply:
@tarilabs if you look at line 32 you will find KIE_API_documentationJavadoc pointing to 7.62.0.Final

Reply:
oh i See now, thanks! 👍